packages/autorest.go/package.json

```diff
@@ -20,7 +20,7 @@
     "stop-testserver": "./node_modules/.bin/autorest-testserver stop",
     "watch": "tsc -p . --watch",
     "prepare": "npm run build",
-    "test": "jest --ci",
+    "test": "jest --ci --passWithNoTests",
```

Comment:
Why do we want to pass if no tests run?

Reply:
Because we don't, at present, have any TS unit tests.